### ISSUE TITLE
ONNX: Invalid type 4 at offset 148

### ISSUE THREAD

**Issue by miverco-coder** [lutzroeder/netron#1117]
- Netron app and version: Mac 7.0.2  and web
- OS and browser version: Chrome Version 114.0.5735.133

I am using netron in Mac, using 6.9.9 I can successfully load the attached onnx file

When one of my colleagues tried to open it using 7.0.2 it failed with error:
`Error loading ONNX model. File format is not onnx.ModelProto (Invalid type 4 at offset 148).`
I have also tried to open it with the web version at http://netron.app/, same failure

Steps to Reproduce:

1. Open  the. onnx file attached in the zip

Please attach or link model files to reproduce the issue if necessary.
<img width="758" alt="image" src="https://github.com/lutzroeder/netron/assets/19730615/b41ec1be-e6fa-4073-a95e-282523270fb5">

[scanefnet.onnx.zip](https://github.com/lutzroeder/netron/files/11948604/scanefnet.onnx.zip)

I exported this model from pytorch (torch version 1.12.1)

**Comment by lutzroeder** [comment]
@miverco-coder can you check the file is correct. Opening the attached file in 6.9.9 yields the same error.

```
git clone https://github.com/lutzroeder/netron.git && cd netron
git checkout $(git log --all --grep="6.9.9" -1 --format="%H")
npm install
npm start
```

**Comment by miverco-coder** [comment]
@lutzroeder that is why I added the screenshot, to show that it can be open. The screenshot is taken with 6.9.9 in a mac (installed from the dmg). It is the same onnx file I attached.

I unzipped what I uploaded and it indeed fails to load in my 6.9.9. I have regenerated the onnx and it loads. When I compare the two files there are some differences in byte 21 already.... The diff is hard to read.
I do not know what could have caused that difference. But anyway it seems like a false alarm.
Sorry for the noise
